builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -227,6 +230,40 @@ node {
 				}
 				break
 
+			case "SLSAPP":
```

Comment:
Can we use consisting formatting? The indentation seems to be inconsistent on all these changes. Thanks.

Reply:
updated.